.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+registries:
+  gradle-plugin-portal:
+    type: maven-repository
+    url: https://plugins.gradle.org/m2
+    username: dummy # Required by dependabot
+    password: dummy # Required by dependabot
```

Comment:
Does this work with these values?

Reply:
absolutely, credentials are required in the dsl but this is a public repo 🤷‍♂️